frontends/web/src/routes/router.tsx

```diff
@@ -0,0 +1,122 @@
+import { ManageAccounts } from './settings/manage-accounts';
+import { Exchanges } from './exchanges/exchanges';
+import ElectrumSettings from './settings/electrum';
+import { Settings } from './settings/settings';
+import { Passphrase } from './device/bitbox02/passphrase';
+import { Account } from "./account/account";
+
+interface Props {
+    devices: TDevices;
+    deviceIDs: string[];
+    accounts: IAccount[];
+    activeAccounts: IAccount[];
+    devicesKey: ((input: string) => string)
+}
+
+const InjectParams: FunctionComponent = ({ children }) => {
+    const params = useParams();
+    return React.cloneElement(children as React.ReactElement, params)
+}
+
+export const AppRouter: FunctionComponent<Props> = ({ devices, deviceIDs, devicesKey, accounts, activeAccounts }) => {
+    const Homepage = <DeviceSwitch
+        default
+        key={devicesKey('device-switch-default')}
+        deviceID={null}
+        devices={devices}
+    />
+
+    const Device = <InjectParams>
+        <DeviceSwitch
+            key={devicesKey('device-switch')}
+            deviceID={null}
+            devices={devices} />
+    </InjectParams>
+
+    const Acc = <InjectParams>
+        <Account
+            code={'' /* dummy to satisfy TS */}
+            devices={devices}
+            accounts={activeAccounts} />
+    </InjectParams>
+
+    const AccSend = <InjectParams>
+        <Send
+            devices={devices}
+            deviceIDs={deviceIDs}
+            accounts={activeAccounts} />
+    </InjectParams>
+
+    const AccReceive = <InjectParams>
+        <Receive
+            devices={devices}
+            deviceIDs={deviceIDs}
+            accounts={activeAccounts} />
+    </InjectParams>
+
+    const AccInfo = <InjectParams>
+        <Info
+            code={''}
+            accounts={activeAccounts} />
+    </InjectParams>
+
+    const BuyInfoEl = <InjectParams>
+        <BuyInfo
+            devices={devices}
+            accounts={activeAccounts} />
+    </InjectParams>
+
+    const MoonpayEl = <InjectParams>
+        <Moonpay
+            code={''}
+            devices={devices}
+            accounts={activeAccounts} />
+    </InjectParams>
+
+    const PassphraseEl = <InjectParams><Passphrase deviceID={''} /></InjectParams>
+
+    const ManageBackupsEl = <InjectParams><ManageBackups
+        key={devicesKey('manage-backups')}
+        devices={devices}
+    /></InjectParams>
+
+    return <Routes>
+        <Route path="/">
+            <Route index element={Homepage} />
+            <Route path="device/:deviceID" element={Device} />
+            <Route path="account/:code">
+                <Route index element={Acc} />
+                <Route path="send" element={AccSend} />
+                <Route path="receive" element={AccReceive} />
+                <Route path="info" element={AccInfo} />
+            </Route>
+            <Route path="add-account" element={<AddAccount />} />
```

Comment:
routes like `/add-account` could be `account/add`, same for `account/summary` (or just `account`)